Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -232,12 +232,31 @@ void ProcessLauncher::launchProcess()
     if (!process.get())
         g_error("Unable to spawn a new child process: %s", error->message);
 
+    // Here we ought to set m_processID to the pid of the WebKit auxiliary
+    // process in the pid namespace of the UI process, but this is probably
+    // impossible to do correctly for Flatpak. Until we're able to fix this, set
+    // it to 0 always to ensure that Flatpak users do not get worse behavior
+    // than non-Flatpak users. This will prevent us from accidentally adding
+    // more functionality that works for developers but doesn't work for users.
```

Comment:
It's already broken for non-flatpak users too; the m_processID refers to the bwrap process and not the web process as intended, so e.g. we suspend only the manager process while leaving the real child process still running.

It's probably easy to fix that for bubblewrap sandbox, but I haven't tried.

Reply:
I went to check and I am disappointed that `bwrap` does not have an option to forward signals to child processes 😢

Reply:
We could add one easily enough, but it wouldn't work here anyway, because the signal we try to send is SIGSTOP. Sending the signal would inherently stop bwrap from forwarding it. :)